From f786d7e548e98209cff36bed3c366f4dc1acd602 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Tue, 27 Oct 2026 12:09:01 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE from keyservers/WKD:
same certificate(s), with self-signatures our copy was missing; the
commands are in the spec comment.
Release bumped with a changelog entry.

Assisted-by: Claude Opus 5.5
---
 B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE | 340 +++++++++++++++++------
 mingw-libidn2.spec                       |  13 +-
 2 files changed, 264 insertions(+), 89 deletions(-)

diff --git a/B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE b/B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE
index e28c60c..14acac7 100644
--- a/B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE
+++ b/B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE
@@ -1,94 +1,262 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
-Comment: B1D2 BD13 75BE CB78 4CF4  F8C4 D73C F638 C53C 06BE
-Comment: Simon Josefsson <simon@josefsson.org>
 
 xjMEXJLOtBYJKwYBBAHaRw8BAQdACIcrZIvhrxDBkK9fV+QlTmXxo2naObDuGtw5
 8YaxlOvNJVNpbW9uIEpvc2Vmc3NvbiA8c2ltb25Aam9zZWZzc29uLm9yZz7ClgQT
 FggAPgIbAwULCQgHAgYVCAkKCwIEFgIDAQIeAQIXgBYhBLHSvRN1vst4TPT4xNc8
+9jjFPAa+BQJq18y3BQkPYcIDAAoJENc89jjFPAa+570A/jRE5zhtTOrXPFTjdPvq
+tY/7qO+Yo78Jo96vVLZ9kw61AQC4wpDdf+o/vku2xskbzZNpoiSDZQzkzD6mnRQ5
+88noCsKWBBMWCAA+AhsDBQsJCAcCBhUICQoLAgQWAgMBAh4BAheAFiEEsdK9E3W+
+y3hM9PjE1zz2OMU8Br4FAmnh9ZEFCQ5r6t0ACgkQ1zz2OMU8Br6FYgD/WVC+sl4a
+VCrnSU/q2b30PXMPLsnW1DeFrjHQnHuuQVMA/2GlQq5O+bqbV9fKJlVhmW7b4h38
+OrcIgBm80X5cSvYEwpYEExYIAD4CGwMFCwkIBwIGFQgJCgsCBBYCAwECHgECF4AW
+IQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCZ9F0JAUJDWRm8AAKCRDXPPY4xTwGvgbS
+AP4ke9wnwbqxvcvKQEod3GxjjB6Q2XzALaLLuo26TuvRhwD+Ke5ee3folbctlUDL
+Mft0wkgL2ZKa+D/xbk82fCWXvAbClgQTFggAPgIbAwULCQgHAgYVCAkKCwIEFgID
+AQIeAQIXgBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJl/YgIBQkLehFUAAoJENc8
+9jjFPAa+CboA+wUa06RD5e5VTCxvSWtPS75Wq2qBeYGZnf0jvUMxa2n4AP4xkUeA
+PPnNuMsTm2fsFCDIGaEM2Yn6Vb2huzzT1Fw/BMKWBBMWCAA+AhsDBQsJCAcCBhUI
+CQoLAgQWAgMBAh4BAheAFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmTyc40FCQl8
+aNkACgkQ1zz2OMU8Br7r3wD+J8GWC7Wt1KfiShDoTwTPVeF9mpkwDNs1bwOhDYFV
+JW0A/1tN1i20gTWPmkrz27T4kzFya+Hwfc/ee0ysRanZ4/MFwpYEExYIAD4CGwMF
+CwkIBwIGFQgJCgsCBBYCAwECHgECF4AWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUC
+YzxUhgUJCHby0gAKCRDXPPY4xTwGvtUgAQCd8YrRkfJmSAaOkGrCzt2ilyCbGcQR
+yxQUoF3BhH6OowEAtesKlEoE43OeFp7RISnLsb4xKnsmZc0pJYGjfMjPFw3ClgQT
+FggAPgIbAwULCQgHAgYVCAkKCwIEFgIDAQIeAQIXgBYhBLHSvRN1vst4TPT4xNc8
 9jjFPAa+BQJiOJ5MBQkGwpOYAAoJENc89jjFPAa+OVQBAIltTRSXhbbx9WSt7RNz
 M4u/1zwc4Ihpqoexu5w42xQoAP937CR37xoIUTyoKSvNeXnJSVkUOeRiCImuEGkD
-3OZrAsKWBBMWCAA+AhsDBQsJCAcCBhUICQoLAgQWAgMBAh4BAheAFiEEsdK9E3W+
-y3hM9PjE1zz2OMU8Br4FAmD3wOsFCQXwdDcACgkQ1zz2OMU8Br6+0QD9GfOxBO7y
-V7CHuG8WPUrkHTEv9FmgPDHR1EoEPOY4B1wA/RcLfvKb0wZrr4I8h0W9MvaO533i
-BDN0TQr0KW0AH2kHwpYEExYIAD4CGwMFCwkIBwIGFQgJCgsCBBYCAwECHgECF4AW
-IQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCX+IpbAUJBI5luAAKCRDXPPY4xTwGvtCJ
-AP0RI2UsT4uZawdEdyUz8yp82/gFflvugIQTX3lJo3iohAD/bFqoYSG0czpfe0lb
-oSz47MF+9KSs+/nx6oGMuAMewgbClgQTFggAPgIbAwULCQgHAgYVCAkKCwIEFgID
-AQIeAQIXgBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJezg00BQkDekmAAAoJENc8
-9jjFPAa+7QMBAKyq5ZypvFOXgcwlNtQdf6F+SP9LnCNSreQRYo4RxSwAAQD7A+O5
-6xFPB1DIM74lpvaExNJFHbJXCIfFGifJycR0A8KWBBMWCAA+AhsDBQsJCAcCBhUI
-CQoLAgQWAgMBAh4BAheAFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAl2wR/oFCQI6
-PUYACgkQ1zz2OMU8Br6FEgEA7veg+Di9SEhfwG2qShdiKwVC9aZqp9gnLIIm6qLt
-fMYA/i0B5GsPglC7upw2a5qyTuSev7uaDLueBSV+mMq8cmAOwpYEExYIAD4WIQSx
-0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXJLOtAIbAwUJARzEAAULCQgHAgYVCAkKCwIE
-FgIDAQIeAQIXgAAKCRDXPPY4xTwGvpqwAQDPG0khoyBLa4gASbXlKAPxgFiflZy6
-EeOJHOUJkMYSrAEAjXUcy1wrw72CADiG5UUXrbBnRSNM3VwPxMvzQNAfcgDOMwRc
-ks91FgkrBgEEAdpHDwEBB0C8whXBx6ys1Ujww2q2T2Kp/PR+Uz2scHDkYEYPuA2o
-BMJ+BBgWCAAmAhsgFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmI4nlsFCQbCkuYA
-CgkQ1zz2OMU8Br6mnAD9HvUcBNa1S1G/33f7uhqn9JDHnGGqFoA2iIkdtoOO5mIA
-/1n/ThshksmDE5Isuea3X92eqSga3+JeJlrVneMnqa0Lwn4EGBYIACYCGyAWIQSx
-0r0Tdb7LeEz0+MTXPPY4xTwGvgUCYPfA9gUJBfBzgQAKCRDXPPY4xTwGvjLnAPoC
-iYpPmCcVKXoEmGBOFNcs+VzK/toEs6Vy0uFcaRJbiwEA/LLa/nZuEnjPRYIFCH7w
-41urLTtcC+SAW4KJyT8dQQ/CfgQYFggAJgIbIBYhBLHSvRN1vst4TPT4xNc89jjF
-PAa+BQJf4imCBQkEjmUNAAoJENc89jjFPAa+dtIA/0DjkqcuxttFDfSROXHQTLde
-0hJebfQSxuXlPu9cYBrfAP0YCUov9QgHqfxH6/nbw6tbdUqKkB1mBfWefZdjr4a3
-D8J+BBgWCAAmAhsgFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAl7ODUcFCQN6SNIA
-CgkQ1zz2OMU8Br4MQQEAsh6/JxnmqDgJoTzzhgzmzqGuvw6bPlgaZrxcPCf2eQAB
-ALRP6cqR4rDB5heHMuUv5B1cy2b38WkwJRqraGRh0oQGwn4EGBYIACYCGyAWIQSx
-0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXbBIDQUJAjo8mAAKCRDXPPY4xTwGvu0SAQCK
-oSTVaP0+CerOAY5GTYs8A8tYiOo94pAWpA0FLOZzpwEA04K55KFRBu59exeb1yGH
-tXSJnQJCXrLnWOFpIwFQHADCfgQYFggAJhYhBLHSvRN1vst4TPT4xNc89jjFPAa+
-BQJcks91AhsgBQkBHMQAAAoJENc89jjFPAa+5JIA/35ATQ4NJVxFiyZIxfMQQ5oL
-CW2GHbYDUKZehXmzXK6bAP9pVSpU3VDSZITqJpvjtDYSSWwIFOXxxKr8DUNx7BAp
-CM4zBFySz4EWCSsGAQQB2kcPAQEHQOxTCIOaeXAxI2hIX4HK9bQTpNVei708oNr1
-Klm8qCGKwsA1BBgWCAAmAhsCFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmI4nl8F
-CQbCkt4AgQkQ1zz2OMU8Br52IAQZFggAHRYhBKPMnIcLnTEKutTPL1FyKwj+R0Wi
-BQJcks+BAAoJEFFyKwj+R0Wip0wBAKAfjlqx5mAivAo69Q4D1B+yZ4TwZkdg24Ue
-Yz0URBLoAPoDjHlpi+hjqWj3ymC3DYc8OGZwJcUi2NcKtXgDQaYkDhASAP9Kz68w
-Qt3DEyIW4ktaI6eeysvM93dHwgs+p8/j8kic7QD/bXTcLqs0J+tfTj/tN4CW6PcJ
-kx3dDJfNvGAY38vVUgzCwDUEGBYIACYCGwIWIQSx0r0Tdb7LeEz0+MTXPPY4xTwG
-vgUCYPfA+gUJBfBzeQCBCRDXPPY4xTwGvnYgBBkWCAAdFiEEo8ychwudMQq61M8v
-UXIrCP5HRaIFAlySz4EACgkQUXIrCP5HRaKnTAEAoB+OWrHmYCK8Cjr1DgPUH7Jn
-hPBmR2DbhR5jPRREEugA+gOMeWmL6GOpaPfKYLcNhzw4ZnAlxSLY1wq1eANBpiQO
-JVIBAJjPGgVSATrdkZIeKqhDNgwBp2sHHiiZW7N0wEw64tm4AQCie9uKfAIE2h+L
-eGVWZ5lfqoXNvhoE/8YWsNVdysQ7DMLANQQYFggAJgIbAhYhBLHSvRN1vst4TPT4
-xNc89jjFPAa+BQJf4imHBQkEjmUGAIEJENc89jjFPAa+diAEGRYIAB0WIQSjzJyH
-C50xCrrUzy9RcisI/kdFogUCXJLPgQAKCRBRcisI/kdFoqdMAQCgH45aseZgIrwK
-OvUOA9QfsmeE8GZHYNuFHmM9FEQS6AD6A4x5aYvoY6lo98pgtw2HPDhmcCXFItjX
-CrV4A0GmJA5yiAEA6Ee/ptArOu2oGc2Okqf38MfN9q7SFTnzd5vutAI4Ib0BAKho
-dAz8j2HMUpxNVGjPHEwco7F1fkkR/78GJFsg+qADwsA1BBgWCAAmAhsCFiEEsdK9
-E3W+y3hM9PjE1zz2OMU8Br4FAl7ODUwFCQN6SMsAgQkQ1zz2OMU8Br52IAQZFggA
-HRYhBKPMnIcLnTEKutTPL1FyKwj+R0WiBQJcks+BAAoJEFFyKwj+R0Wip0wBAKAf
-jlqx5mAivAo69Q4D1B+yZ4TwZkdg24UeYz0URBLoAPoDjHlpi+hjqWj3ymC3DYc8
-OGZwJcUi2NcKtXgDQaYkDuQiAPwKnKAbzegaSATxN1cdFia4m80uJNFHMQL679WS
-BG3FIAEA8uLgxGud6SqFgIaFR4wrzrIgzVWqHxDuu56fJSf/iAfCwDUEGBYIACYC
-GwIWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXbBIEgUJAjo8kQCBCRDXPPY4xTwG
-vnYgBBkWCAAdFiEEo8ychwudMQq61M8vUXIrCP5HRaIFAlySz4EACgkQUXIrCP5H
-RaKnTAEAoB+OWrHmYCK8Cjr1DgPUH7JnhPBmR2DbhR5jPRREEugA+gOMeWmL6GOp
-aPfKYLcNhzw4ZnAlxSLY1wq1eANBpiQOte4A/2ju4S1HrcdnAVyd+BK5ieldpgOJ
-qKp9EFujf+zKuoKsAP4uq8+vBc6QmVri+ExSbqT/q5/pcUcBkxBVdda6G++yDMLA
-NQQYFggAJhYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJcks+BAhsCBQkBHMQAAIEJ
-ENc89jjFPAa+diAEGRYIAB0WIQSjzJyHC50xCrrUzy9RcisI/kdFogUCXJLPgQAK
-CRBRcisI/kdFoqdMAQCgH45aseZgIrwKOvUOA9QfsmeE8GZHYNuFHmM9FEQS6AD6
-A4x5aYvoY6lo98pgtw2HPDhmcCXFItjXCrV4A0GmJA4KLgEAuT/fZL+h/jy/QW9y
-fSi4BT56PGU7n/MoUzjOwG8flj4BAOCm+fqLhrM0SQzzjA0qdCrsTK/7rBSZ908t
-bNYOawgOzjgEXJLPahIKKwYBBAGXVQEFAQEHQDGVG6YNbqdljsD5UJ96T1WkTHo8
-mZ0nJg+3eU0cx3ocAwEIB8J+BBgWCAAmAhsMFiEEsdK9E3W+y3hM9PjE1zz2OMU8
-Br4FAmI4nlQFCQbCkuoACgkQ1zz2OMU8Br7QXgEAhd81O+aan2sLslGjC749qnaq
-AHwan9Nxz8f+J9SzG3UBAPTMOR4aoC+u8KxUxkNU53Ccf+bgGHY4OJ0gTDLLpPIG
-wn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCYPfA8QUJBfBzhwAK
-CRDXPPY4xTwGvtQ/APwP8XxDUEogiXWIatg4A8oap242CJQjPf9IrwETGf7ckAEA
-oF/XCoJNOsSKRGPgTnVA7vD8tNQpv6BZsk4WB3ShOAfCfgQYFggAJgIbDBYhBLHS
-vRN1vst4TPT4xNc89jjFPAa+BQJf4il7BQkEjmURAAoJENc89jjFPAa+410BAOK0
-ISCDC4Rd/LLUBMSfZsLIpI1yuFnDPp+OAc0Ec9tkAQDJ1Jz/XlxCwf/gYoqTLXkZ
-xGzd33QWjH2JKVZyKNF2CsJ+BBgWCAAmAhsMFiEEsdK9E3W+y3hM9PjE1zz2OMU8
-Br4FAl7ODUAFCQN6SNYACgkQ1zz2OMU8Br7vEAD/S59d+oS7f8i38eCmIwcTwHX3
-faEWHd8alhQcTWZp/gYA/12a4TWmNzBAVeMiMeLbOiU+3SX1xnGzY8StU9gZV3UB
-wn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXbBIBgUJAjo8nAAK
-CRDXPPY4xTwGvuhAAQDvbSbrxBwmqvbEwJuprPVn9Nx/bwfWKfbS2Dqg/huOhAD9
-E1YBmxr5qiAbFJKP0O7Sj22HXX7LiALyf8eORehJPAjCfgQYFggAJhYhBLHSvRN1
-vst4TPT4xNc89jjFPAa+BQJcks9qAhsMBQkBHMQAAAoJENc89jjFPAa+4kMBAILG
-9rwrhcPSAvay4XDgJ8SqDHm8AsKA3E7zirxVq5X9AQCNlXB+RWh7yX61utkJrziB
-qQti/j3vHKX6c3yU4FLNCw==
-=KvZq
+3OZrAsKtBBMWCAA+AhsDBQsJCAcCBhUICQoLAgQWAgMBAh4BAheAFiEEsdK9E3W+
+y3hM9PjE1zz2OMU8Br4FAmD3wOsFCQXwdDcAIQkQ1zz2OMU8Br4WIQSx0r0Tdb7L
+eEz0+MTXPPY4xTwGvr7RAP0Z87EE7vJXsIe4bxY9SuQdMS/0WaA8MdHUSgQ85jgH
+XAD9Fwt+8pvTBmuvgjyHRb0y9o7nfeIEM3RNCvQpbQAfaQfClgQTFggAPgIbAwUL
+CQgHAgYVCAkKCwIEFgIDAQIeAQIXgBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJf
+4ilsBQkEjmW4AAoJENc89jjFPAa+0IkA/REjZSxPi5lrB0R3JTPzKnzb+AV+W+6A
+hBNfeUmjeKiEAP9sWqhhIbRzOl97SVuhLPjswX70pKz7+fHqgYy4Ax7CBsKWBBMW
+CAA+AhsDBQsJCAcCBhUICQoLAgQWAgMBAh4BAheAFiEEsdK9E3W+y3hM9PjE1zz2
+OMU8Br4FAl7ODTQFCQN6SYAACgkQ1zz2OMU8Br7tAwEArKrlnKm8U5eBzCU21B1/
+oX5I/0ucI1Kt5BFijhHFLAABAPsD47nrEU8HUMgzviWm9oTE0kUdslcIh8UaJ8nJ
+xHQDwpYEExYIAD4CGwMFCwkIBwIGFQgJCgsCBBYCAwECHgECF4AWIQSx0r0Tdb7L
+eEz0+MTXPPY4xTwGvgUCXbBH+gUJAjo9RgAKCRDXPPY4xTwGvoUSAQDu96D4OL1I
+SF/AbapKF2IrBUL1pmqn2Ccsgibqou18xgD+LQHkaw+CULu6nDZrmrJO5J6/u5oM
+u54FJX6YyrxyYA7ClgQTFggAPhYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJcks60
+AhsDBQkBHMQABQsJCAcCBhUICQoLAgQWAgMBAh4BAheAAAoJENc89jjFPAa+mrAB
+AM8bSSGjIEtriABJteUoA/GAWJ+VnLoR44kc5QmQxhKsAQCNdRzLXCvDvYIAOIbl
+RRetsGdFI0zdXA/Ey/NA0B9yAMJ1BBAWCAAdFiEETRQFBlOkAtc2hwSdJATJVG4U
+U2AFAma2uakACgkQJATJVG4UU2DpzQEAtGkZNbKAsd6UyPANoVoj/vz+hfMNOyQ0
+AQx4VrKfz/YA/A0qexJ4s2jc+Ew0/8AifWgwK0duMyKC4SuFsRAOtVYJwsFzBBAB
+CAAdFiEErkjA9ZsZmKBikqaZlr1P9k5wn94FAmQQJKwACgkQlr1P9k5wn94r9A/+
+IhoY0u453w8edPPFZpujpce+xZ9jdYZZ5vTJmJOjNbS8ELW9z2FEkFCBIrSeEz+2
+NPV2jvSelbMoTELUFIs4dCSLTNtJb8Vl2YMRiFh8WtJDRCc7qifGDt+QTknQBUOo
+FVYolwogcHXMx+veE0cEktxhlG8pIRs4rRV0dlJmvquQ7F2pwsq+d614Z9fzsUHK
+zJzm9Zz7ScYGHXz5Nw7DMwM1qp0nNzTbArbx3u9s7AvYBXbEthJZefCpRxYBroYJ
+zCGR6ZlWgydd9onwRgTpB3W09a/0FbL9Xsr1ZBbbLArWQx6wL6NQ3f/wHK3En58i
+Cu41UeZeHmFVYklcDe4L1K/ZOC9b8mEy5DGlme3efnJLyWUiVWiKjYNn6KF5At2L
+ctnd08VvSGRxY3QpLpY5VcDjNTEVSFeqIGnRmJX37esUFYf9K6XCcQeStf78jK9R
+yF9dZ/JDBKCxRLohNWel2saOJd5AFmzA+rib0RJDbxxnN+M6J/bW65tsWagfLKJM
+q2VWGpTF/MGAxr5YolxXD6eVD2ngcoh0yauk7Cj1c2Xz22+0irhuDDS+PK9rs2I2
+72BBvgPnYtrSS0MJ0QTXK7TUpgwxc2CEzLdj94knzRPZfr3SGA/VYeRrDCApFYkV
+jmrgvQOLcuQFQV3JXfvnZsAOaI9yZ77k6tRa55OyWM/CwXMEEAEIAB0WIQRGIiXD
+tG80h5/ISWzWBYSO1+aYcQUCY05WcgAKCRDWBYSO1+aYcU1GD/9K2d/wBTyfWLJl
+OmTsyxuy+w6hGLJYV/CgpW6RMFBEM2rZzgxvZzzTuYXmYT9RIxf8PIutLUetR4gX
+j9WQ8GmGb4j6uh4nU3FN9Hq6TIQfeZ1n3wYYFdH6wDuH3lglhZkBV2Edxof78ueh
+u+yADg8hqEf61ElnVkXduj7GZp3qlSr/16+XG813xZ5uZWIUbQq6KaFSy3yrYEvv
+PqMvLjjMHsFSoC9gqTCREmhpxGTy4hxcsOmNA331Icdop2NE1lYg5Nc6REaxE6yE
+f3lwvXgTW6oohEdjtI9gy5D4EEtuZBXUWOoeoIJDsgAMxCyEtnE9QArgvlSBKZho
+8iYaREgNBCs7C1aNLw/VqbMGOGN4HeiTaOwGNffwD5gWlh4GVw926m/+53+R1ZGW
+DGUvno7j6o3mzOv116F8zoKNypUvAGhYZWby40oRG0Zm9fdu+TRaJMcCLVWqKew3
+KfSGIW47cFw2sGRbdJY3TQv1b4mQkrg5X732k46DUKhQMLRlrK3vPPUqCYdYV/9H
+p3+03WRypfdS1A+udBEAu0MXZXpo606BdCm3g1B8FGS82XIlb9C1txJMzWHYRb2V
+obufPn0t7Q4+vtao9kxfF0xeeGZxOyfiYqX0ZFzaa2nQ5uVykml+1aYHjCY92tY+
+eiCG/AICq6jR5lw7bL8ZmLdaWqe+gsLBcwQQAQoAHRYhBI+IOB3VyHPkrkh9pRmb
+8SQ2MgRqBQJjO/2GAAoJEBmb8SQ2MgRqB1cQAI/iqVCIFI1/gICTTF4TLFp7Puzx
+ojjxJMWnJHYPV4/CA/XtV3DHxM5SkEy9OXsKg+d6nWRQZVF6Et86E/i/EaYw0p5W
+FtZAndnVwFDmARRQbw6O2LsISvc5ttaxkh06MVzyT1TCtaqC9EoYambxz9BLEh7b
+4IQ5i9w8Xl35gh449EXBsbs7Ku/h6pwZyD5A1yfZGJyaE2w6hK4RXoildgx1nmNt
+zl0zMbYhEbDzMHROins8ImqF4WLaWRiAaQ5lGosgtm+n0rKBReO4zWMDjzFYu2dL
+J/K384vhxHaInQNssCrdXzYoYmg/9QSJqapFH+8uj+iwq9HE6sx+gNX8VfvyWELo
+MCtAf3ouDXn3l2E4a6YjgX9FRagr17m8X3kqUpCFh/NqHSQRSRZCmowAF8APTF4w
+JA4qwlx0naqo6DIR0QPc5WdTioB3fHb0g13dXHGXpem+5E87PzwW+RxEACrC7LcN
+0snKE7tuAeTkVg/F0vcEXPf3Xl2Es7QqS8Aog6sLS8TDRGHgJkoAljIEQeVGeTQn
+fz8WZ8aIsmYR46cfcKX2WK5uEFnwRN24EJ+TMjTaCwSx7UjrSJq3BeVQPd0lhRZ+
+8AFnike7OU8iNldKx7A1bAVHRxetA3AEveRksfp2gksZVlaeq4NBLLtwr5pgde6z
+TY6/evVS/3dNk91mwsFHBBABCgAdFiEEmqm9sRuxuZohKFozBmSnaVQmXowFAmCt
+Nd8ACgkQBmSnaVQmXozT4A6dFyIBhVNK5SYmbIHhDTrAiZtXm+RUM0+P8lzyqU2C
+f34dyBNr9rqYuMm6Gnkdkkti1UfYLhus+CcsY08dVt5h/q/2C3Bvjevoe8BuC+oY
+GDWcEcv+5rhvhYVEKzTCiKXH2P0LKxXHb2xX6FJ7UQf2lxTybpJ8uH7Qahti4ak9
+Ku9+8DzwtF6p6/+BWKoQ9D3V2bhGUMG9j3UHZ9ItWwPt4xvelqjsJqdyDjJ7QGq2
+mDM/7bwaCdPuh3hgv+RdR3r5zUR8qo2mUEF3Zrr0XbPePdru0603DrLaOV1G4kSg
+Df+G/VtbykyqgsvGRV48rchDGu/OyXu57qpKsY2CLDdApOqztbvKKj+QSI433gTZ
+Ie+QRUo1d8EMMfPrXcPTl3g6WkRXMDth9828ItkcDDqs2ZKKrf7zYbvE+l+GPLux
+pl766B6QPP1xxtWb7ENW6ozYraaC3BDVAcFS29hsvfqxMwcFaaYCTQyVt+0P1MUn
+V5oTZOfi7PL/efHBg5rrLV56keNyYRSLlEGopsiNf6qx7nSidYTcAfCPGyXgw9np
+pJ3zj2qpOeohDsvwiqOFBFj6lYSz5spztosAyk+5A9H/qt304gPPCoYdJxSyUjs/
+5mqvhEN2wsFzBBABCgAdFiEEdjYv07yrl3cqbcFawY7miuGQ0LwFAl99p/MACgkQ
+wY7miuGQ0Lwt5Q//SXbXwxklV8op8H25TVifN+vFCJl5QU6npi91VfZTjZnkE5JR
+IVL69Mor2cMHCyL3HfRJCleubJ6lE9JLp71Tjk5QhwNTvK4a2e4uG8R1VtTO8kLa
+RLMHQRPJ0e3BuTozdGuTzqYGiWzeT6tif8twqjCL78/EVtRsdlAGsQl22kvDfjlb
+R4xvaEfft6l9HeYWORZqfMbO4oJY5Qgn/yhohTF3DCu4l62Se6lvfhxZSy8jC/2h
+HwcZbbrNNjFiaPznKEH3Sbf+rkjnNFe+06ZI/21kNpRluufeZzMGvj9BJFC+bpRb
+EXh/AeA+oZA46lTXSJjmnu7CYXdgaQiKjH4mVRpAiN+e3fetciu9meUWQX+5TNFi
+keFEB2geIf87QNRPt/DbftHC0qjF6m85lMlfypq3PKIeSWKqOah7OdP9KROH1O4M
+NlEx6AmMDpeNuW0v4HYMsTNsCPu48e1DYnXqBi2XWvPDBt0jyh8rIOK0n6pmHSut
+D7tCuIzOkaZb2h0GCIf9ix1ekXPC4Vzax/AZrUXEQwokZL3VezYVlcVoVQnSfALZ
+KiUljLznmLSOWeG/qDgkCuKFdFpULfv19tmbCnzQ+4/VaJ9YHlts6QyQ2y6ImdUZ
+bH+SLotELgtrsmgJNfi2sbYmRIfJuLcVO95p8xlIYFTcizM9sKL26ogoY73CwXME
+EwEIAB0WIQRiCqQisS0eHbXzPDIZDl9rP1fDhAUCX3tyGQAKCRAZDl9rP1fDhCqk
+EACRehYCPxS49yFW4QO3nLyzpp52lu17/9Gzto+s0q5HvvCtNXVy1CjH3kw6T2yF
+2V1/F8YATEhzvM10Z+aZAuEvZYwlLnCfH9RDRbAOpMrAPlWEKcz727s6O/dDxGk7
+eMFPkmji3PwOrZE3K9lQIO2Cyajz+j+1xdE/DRZofTSwI4jLaupTqN95+cOy41wu
+ANby4wc6w6FG/P+oIppQQB4p1s7pV5hfPAdNVLa7mLAWC5ABuGoJKkhKgKXnP1Ms
+UEykP0y+1WkDOzDj1Mwwibj3Z9KrBZBMxUuH3iaHRfn64asdofPXe+OzE9k3Rrwm
+0TznUswu/NIiIIDLgS9AulYRAdmbqrEQcZoqwfULSb//iub7YwXSat8kNRdHID4i
+oCHxMdSoSOTmzHrjn+U0wtJAQpFfHn4OoWPuBXWuhE02w1eb8y6yMPuLcdQLQGQW
+yf5CzmD9OkioM5iIQdXB/xWxsp7wPb7KhpzF/JsgdMIibpbWLlqQ3CwDAtPoocas
+EPi611LYc6p8P4B0dExhRrPu1CHRy6MXiAFO8baAutJoTUvOzcMpEzSA6q9RSE+o
+1ktjF60xQ3TQd7E3HW2Kl2Zxsxv1B7/Uy6HC4Je2dyEO4FQHG8J7hFGvVmvts+Hf
+6vI2isk58VMDLvlk56wTlWNgDbDWM7u6H8aVdKBehZ5EDsLBRwQQAQoAHRYhBJqp
+vbEbsbmaIShaMwZkp2lUJl6MBQJck+zBAAoJEAZkp2lUJl6MQK4OnRQia+hPpPen
+SZ0UMQPuMdMi0U77VNWGYsWoz4diVEni1X7p6q9Lp6RrnMyP1qAVuvgyzCDmKow3
+nE+vR626Eh7iqnw6L3iwWYj0Xg8ebZsm8mb3HdmuobhMSoqudLllxmzNViIzpSUM
+4GyuvPK0hd3/I6FO6CNUklT8N7WsDHzSCPvmdaDMyzVSGANWl5rnfXW/qPF9sALT
+P07JSXWFN8SCOrJCXSPGAWOF0iSeRHbcwLahgof9nY11D7OewNvRlitg/0DEkjhK
+WJ6fY1k5R8ryHWtQJsB4jJ6uq9tuJ6EKfc/A85eBJjpJdJpe9AW+RA4FHPEFcfKh
+B4xWB1J2HB/0ocs5m6leW1WuIXzDc9WB96NIkSQWtsnqUfxin443/rIf5muS373G
+yjmR87VzDx8V6LIitSoDB0yBaNW5+jwlt7LotnwZQeEnRkS4VDkz4jh22XzdCbmp
+mdqyCDgTCuyChfDp2EjDFg+h2bV3xEvDZI7rx15gkfB9PDqkWbfUBE3VkakQWDlQ
+mNs409F3eJRB8kaGVlopugZS0iPwx1Fwo+t78yoa824VMavHCpw2k5QwZr+L2Y9e
+83zAhOWfrs9U6x93b4j+MxDXIOQQJu9REyFCQs4zBFySz3UWCSsGAQQB2kcPAQEH
+QLzCFcHHrKzVSPDDarZPYqn89H5TPaxwcORgRg+4DagEwn4EGBYIACYCGyAWIQSx
+0r0Tdb7LeEz0+MTXPPY4xTwGvgUCatfLugUJD2HAQgAKCRDXPPY4xTwGvmxtAQC1
+829d8TCq2qGG9a4WT7SQLEcTEEgE5ZyF3BoYBYFjNAEA+7PYO5DTUwYSjbPeCxn1
+SD7jG5uUB2OA1GWJU4HzsAbCfgQYFggAJgIbIBYhBLHSvRN1vst4TPT4xNc89jjF
+PAa+BQJp4JbXBQkOaotiAAoJENc89jjFPAa+tjMBAMGW7OGP2IoA13hvD8PvWpVZ
+AyavGjp2O+UW+6bd5q1zAP9vSRbpoFiYtKZl0BjF1fJjeMb4qEFMcDvP6wLzhDjD
+AMJ+BBgWCAAmAhsgFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmfRdDUFCQ1kZkAA
+CgkQ1zz2OMU8Br6PsQEAphp43m/5TYJgBMOLux4yWrUev1Oyc3YrCSn9IETuZooB
+APJAJTHoPf2EsuWLZVGeNWsEulgG6CYl6Xb5/N+1bP4Dwn4EGBYIACYCGyAWIQSx
+0r0Tdb7LeEz0+MTXPPY4xTwGvgUCZf2IJAUJC3oQrwAKCRDXPPY4xTwGvoxCAQCe
+/iMQZvHZmSQef5RnL1HOWy03OHtsZyhGLnQjsx7PhAEA3O2K0dNbPW2iZMcn9MXA
+Odmff3zkfNrWEWkZR/x5XgzCfgQYFggAJgIbIBYhBLHSvRN1vst4TPT4xNc89jjF
+PAa+BQJk8nOXBQkJfGgiAAoJENc89jjFPAa+J88A/3+YlolaZ6XOlgRQAfrF+kvy
+actogaMs/6h5QXBdyYvTAP9BwWS7qNwu0/dBkrvhUiH4OVaH533hEt0jb7Z3F9EE
+CcJ+BBgWCAAmAhsgFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmM8VJQFCQh28h8A
+CgkQ1zz2OMU8Br4NsAEAwzWeBs6p/lbMw9OOkKRLlNJbFpMeCzMWlEYVnzUkE1AA
+/3vq/A7fE5a+GaU2HzYks7bKr2ZQVRcxQQadisPwGA4Iwn4EGBYIACYCGyAWIQSx
+0r0Tdb7LeEz0+MTXPPY4xTwGvgUCYjieWwUJBsKS5gAKCRDXPPY4xTwGvqacAP0e
+9RwE1rVLUb/fd/u6Gqf0kMecYaoWgDaIiR22g47mYgD/Wf9OGyGSyYMTkiy55rdf
+3Z6pKBrf4l4mWtWd4yeprQvClQQYFggAJgIbIBYhBLHSvRN1vst4TPT4xNc89jjF
+PAa+BQJg98D2BQkF8HOBACEJENc89jjFPAa+FiEEsdK9E3W+y3hM9PjE1zz2OMU8
+Br4y5wD6AomKT5gnFSl6BJhgThTXLPlcyv7aBLOlctLhXGkSW4sBAPyy2v52bhJ4
+z0WCBQh+8ONbqy07XAvkgFuCick/HUEPwn4EGBYIACYCGyAWIQSx0r0Tdb7LeEz0
++MTXPPY4xTwGvgUCX+IpggUJBI5lDQAKCRDXPPY4xTwGvnbSAP9A45KnLsbbRQ30
+kTlx0Ey3XtISXm30Esbl5T7vXGAa3wD9GAlKL/UIB6n8R+v528OrW3VKipAdZgX1
+nn2XY6+Gtw/CfgQYFggAJgIbIBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJezg1H
+BQkDekjSAAoJENc89jjFPAa+DEEBALIevycZ5qg4CaE884YM5s6hrr8Omz5YGma8
+XDwn9nkAAQC0T+nKkeKwweYXhzLlL+QdXMtm9/FpMCUaq2hkYdKEBsJ+BBgWCAAm
+AhsgFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAl2wSA0FCQI6PJgACgkQ1zz2OMU8
+Br7tEgEAiqEk1Wj9PgnqzgGORk2LPAPLWIjqPeKQFqQNBSzmc6cBANOCueShUQbu
+fXsXm9chh7V0iZ0CQl6y51jhaSMBUBwAwn4EGBYIACYWIQSx0r0Tdb7LeEz0+MTX
+PPY4xTwGvgUCXJLPdQIbIAUJARzEAAAKCRDXPPY4xTwGvuSSAP9+QE0ODSVcRYsm
+SMXzEEOaCwlthh22A1CmXoV5s1yumwD/aVUqVN1Q0mSE6iab47Q2EklsCBTl8cSq
+/A1DcewQKQjOMwRcks+BFgkrBgEEAdpHDwEBB0DsUwiDmnlwMSNoSF+ByvW0E6TV
+Xou9PKDa9SpZvKghisLANQQYFggAJgIbAhYhBLHSvRN1vst4TPT4xNc89jjFPAa+
+BQJq18u6BQkPYcA2AIEJENc89jjFPAa+diAEGRYIAB0WIQSjzJyHC50xCrrUzy9R
+cisI/kdFogUCXJLPgQAKCRBRcisI/kdFoqdMAQCgH45aseZgIrwKOvUOA9QfsmeE
+8GZHYNuFHmM9FEQS6AD6A4x5aYvoY6lo98pgtw2HPDhmcCXFItjXCrV4A0GmJA43
++wD/TKZwRSwx6JmrwpF80WiRGmKmIoLx5e1u6lrrMIthYE0BAIO8SiBeExxy2Y+H
++PK8BRu1VtrbPKP0mmKSTKIFiZcGwsA1BBgWCAAmAhsCFiEEsdK9E3W+y3hM9PjE
+1zz2OMU8Br4FAmngltcFCQ5qi1YAgQkQ1zz2OMU8Br52IAQZFggAHRYhBKPMnIcL
+nTEKutTPL1FyKwj+R0WiBQJcks+BAAoJEFFyKwj+R0Wip0wBAKAfjlqx5mAivAo6
+9Q4D1B+yZ4TwZkdg24UeYz0URBLoAPoDjHlpi+hjqWj3ymC3DYc8OGZwJcUi2NcK
+tXgDQaYkDrOwAPoCCB6QQKQ6XA49bCocw54Jqb8EH9FZB5nurPhBu6koKQEA3v1z
+K1NHND2t44vOXDKmYb8yThSP7KHuutpcy9EGNwXCwDUEGBYIACYCGwIWIQSx0r0T
+db7LeEz0+MTXPPY4xTwGvgUCZ9F0SgUJDWRmSQCBCRDXPPY4xTwGvnYgBBkWCAAd
+FiEEo8ychwudMQq61M8vUXIrCP5HRaIFAlySz4EACgkQUXIrCP5HRaKnTAEAoB+O
+WrHmYCK8Cjr1DgPUH7JnhPBmR2DbhR5jPRREEugA+gOMeWmL6GOpaPfKYLcNhzw4
+ZnAlxSLY1wq1eANBpiQOwUUBAO64fbZek6FPlRK0DrlWsrjCXuLi6PUxyzCAY6lG
+2nhUAQC6qobB9mkZlZ0qihy1x4JRtflqFcqqT9n7iUZkCDIiDcLANQQYFggAJgIb
+AhYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJl/YgrBQkLehCqAIEJENc89jjFPAa+
+diAEGRYIAB0WIQSjzJyHC50xCrrUzy9RcisI/kdFogUCXJLPgQAKCRBRcisI/kdF
+oqdMAQCgH45aseZgIrwKOvUOA9QfsmeE8GZHYNuFHmM9FEQS6AD6A4x5aYvoY6lo
+98pgtw2HPDhmcCXFItjXCrV4A0GmJA4ZxgD/bphBjTlsu2deIiNOIDOsZCtFfFMc
+8DwC/tMMUe8h61YA/2wdKhTw59PCZOIScbAuWxXtAlyVmDdg3P9ilyn1OpAGwsA1
+BBgWCAAmAhsCFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmTyc5wFCQl8aBsAgQkQ
+1zz2OMU8Br52IAQZFggAHRYhBKPMnIcLnTEKutTPL1FyKwj+R0WiBQJcks+BAAoJ
+EFFyKwj+R0Wip0wBAKAfjlqx5mAivAo69Q4D1B+yZ4TwZkdg24UeYz0URBLoAPoD
+jHlpi+hjqWj3ymC3DYc8OGZwJcUi2NcKtXgDQaYkDq8mAPoD2rY6vX3V/dxmpP/L
+legQkygGhHRGgSJqfXrI5nOWfgEAwYoVw4TDz5dBHMghYdUG3lOs3nNjdS0D4zWC
+fHZ5HwvCwDUEGBYIACYCGwIWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCYzxUmQUJ
+CHbyGACBCRDXPPY4xTwGvnYgBBkWCAAdFiEEo8ychwudMQq61M8vUXIrCP5HRaIF
+AlySz4EACgkQUXIrCP5HRaKnTAEAoB+OWrHmYCK8Cjr1DgPUH7JnhPBmR2DbhR5j
+PRREEugA+gOMeWmL6GOpaPfKYLcNhzw4ZnAlxSLY1wq1eANBpiQOaukA/0gjmfw6
+5eZaic93tHkNOblWJNW8/A5ohLC1TFCfy6gLAQDsxknT9a3CVpYs6sYIRPdjYbcE
+e2BOBCnkOme1/PZhB8LANQQYFggAJgIbAhYhBLHSvRN1vst4TPT4xNc89jjFPAa+
+BQJiOJ5fBQkGwpLeAIEJENc89jjFPAa+diAEGRYIAB0WIQSjzJyHC50xCrrUzy9R
+cisI/kdFogUCXJLPgQAKCRBRcisI/kdFoqdMAQCgH45aseZgIrwKOvUOA9QfsmeE
+8GZHYNuFHmM9FEQS6AD6A4x5aYvoY6lo98pgtw2HPDhmcCXFItjXCrV4A0GmJA4Q
+EgD/Ss+vMELdwxMiFuJLWiOnnsrLzPd3R8ILPqfP4/JInO0A/2103C6rNCfrX04/
+7TeAluj3CZMd3QyXzbxgGN/L1VIMwsDaBBgWCAAmAhsCFiEEsdK9E3W+y3hM9PjE
+1zz2OMU8Br4FAmD3wPoFCQXwc3kBJgkQ1zz2OMU8Br52IAQZFggAHRYhBKPMnIcL
+nTEKutTPL1FyKwj+R0WiBQJcks+BAAoJEFFyKwj+R0Wip0wBAKAfjlqx5mAivAo6
+9Q4D1B+yZ4TwZkdg24UeYz0URBLoAPoDjHlpi+hjqWj3ymC3DYc8OGZwJcUi2NcK
+tXgDQaYkDo0gBBkWCAAdFiEEo8ychwudMQq61M8vUXIrCP5HRaIFAlySz4EAIQkQ
+UXIrCP5HRaIWIQSjzJyHC50xCrrUzy9RcisI/kdFoqdMAQCgH45aseZgIrwKOvUO
+A9QfsmeE8GZHYNuFHmM9FEQS6AD6A4x5aYvoY6lo98pgtw2HPDhmcCXFItjXCrV4
+A0GmJA4WIQSx0r0Tdb7LeEz0+MTXPPY4xTwGviVSAQCYzxoFUgE63ZGSHiqoQzYM
+AadrBx4omVuzdMBMOuLZuAEAonvbinwCBNofi3hlVmeZX6qFzb4aBP/GFrDVXcrE
+OwzCwDUEGBYIACYCGwIWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCX+IphwUJBI5l
+BgCBCRDXPPY4xTwGvnYgBBkWCAAdFiEEo8ychwudMQq61M8vUXIrCP5HRaIFAlyS
+z4EACgkQUXIrCP5HRaKnTAEAoB+OWrHmYCK8Cjr1DgPUH7JnhPBmR2DbhR5jPRRE
+EugA+gOMeWmL6GOpaPfKYLcNhzw4ZnAlxSLY1wq1eANBpiQOcogBAOhHv6bQKzrt
+qBnNjpKn9/DHzfau0hU583eb7rQCOCG9AQCoaHQM/I9hzFKcTVRozxxMHKOxdX5J
+Ef+/BiRbIPqgA8LANQQYFggAJgIbAhYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJe
+zg1MBQkDekjLAIEJENc89jjFPAa+diAEGRYIAB0WIQSjzJyHC50xCrrUzy9RcisI
+/kdFogUCXJLPgQAKCRBRcisI/kdFoqdMAQCgH45aseZgIrwKOvUOA9QfsmeE8GZH
+YNuFHmM9FEQS6AD6A4x5aYvoY6lo98pgtw2HPDhmcCXFItjXCrV4A0GmJA7kIgD8
+CpygG83oGkgE8TdXHRYmuJvNLiTRRzEC+u/VkgRtxSABAPLi4MRrnekqhYCGhUeM
+K86yIM1Vqh8Q7ruenyUn/4gHwsA1BBgWCAAmAhsCFiEEsdK9E3W+y3hM9PjE1zz2
+OMU8Br4FAl2wSBIFCQI6PJEAgQkQ1zz2OMU8Br52IAQZFggAHRYhBKPMnIcLnTEK
+utTPL1FyKwj+R0WiBQJcks+BAAoJEFFyKwj+R0Wip0wBAKAfjlqx5mAivAo69Q4D
+1B+yZ4TwZkdg24UeYz0URBLoAPoDjHlpi+hjqWj3ymC3DYc8OGZwJcUi2NcKtXgD
+QaYkDrXuAP9o7uEtR63HZwFcnfgSuYnpXaYDiaiqfRBbo3/syrqCrAD+LqvPrwXO
+kJla4vhMUm6k/6uf6XFHAZMQVXXWuhvvsgzCwDUEGBYIACYWIQSx0r0Tdb7LeEz0
++MTXPPY4xTwGvgUCXJLPgQIbAgUJARzEAACBCRDXPPY4xTwGvnYgBBkWCAAdFiEE
+o8ychwudMQq61M8vUXIrCP5HRaIFAlySz4EACgkQUXIrCP5HRaKnTAEAoB+OWrHm
+YCK8Cjr1DgPUH7JnhPBmR2DbhR5jPRREEugA+gOMeWmL6GOpaPfKYLcNhzw4ZnAl
+xSLY1wq1eANBpiQOCi4BALk/32S/of48v0Fvcn0ouAU+ejxlO5/zKFM4zsBvH5Y+
+AQDgpvn6i4azNEkM84wNKnQq7Eyv+6wUmfdPLWzWDmsIDs44BFySz2oSCisGAQQB
+l1UBBQEBB0AxlRumDW6nZY7A+VCfek9VpEx6PJmdJyYPt3lNHMd6HAMBCAfCfgQY
+FggAJgIbDBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJq18u3BQkPYcBNAAoJENc8
+9jjFPAa+hwcA/2DXMbXDXT+WFvuSys31PzGcdM5k2Jt3bt2T5f6V5EKRAP45aWVi
+4zU3VkD9rlCPK/umer6GvF6pKD5yebgCuUNDBsJ+BBgWCAAmAhsMFiEEsdK9E3W+
+y3hM9PjE1zz2OMU8Br4FAmngltcFCQ5qi20ACgkQ1zz2OMU8Br5E1QD/Z9pA7+cV
+PToT4yWGVB2j/RyGVBcLlNkw9NW9UEXCgs4A/3EskrSjDkSNPNgkHgrzXQAv8i/o
+8Q4ASADhw9MdWVsJwn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUC
+Z9F00gUJDWRm6AAKCRDXPPY4xTwGvtDNAQDz0au95C5x2DQzJ4lQTs1HYtl3hd9o
+nYxFln478iCVDgD+MGfgC8SHoYrN1MNgcxmROrbV0NkPg3giTt0z6SKmlwzCfgQY
+FggAJgIbDBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJl/YgwBQkLehDGAAoJENc8
+9jjFPAa+phoA/jrDqIrl/55vUMBhIQv+TP635d2iCTEnyFmbUcP9+gh6APoDsXal
+Vd2cOGxQtSC+TF8PkZMn1TLkJKAjVxr+xx40AsJ+BBgWCAAmAhsMFiEEsdK9E3W+
+y3hM9PjE1zz2OMU8Br4FAmTyc6MFCQl8aDkACgkQ1zz2OMU8Br7JeQD8DUWTsvFe
+kvrTb5gFovupKpNJo0G3g+FS9wQ8m7ZZuyQBAIqLSadpCHUtG3FjJS1TxndFz16P
+E5FokCSP6UHXnskBwn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUC
+YzxUjwUJCHbyJQAKCRDXPPY4xTwGvj99AP4mTm069Q6anW3ckEfsFlnNgbLpqNj0
+BJ6+FcJ88VZqQgD/fpfOKMwK14sjwa+m8utT2dI7/0khB55i246+9Qi9FwzCfgQY
+FggAJgIbDBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJiOJ5UBQkGwpLqAAoJENc8
+9jjFPAa+0F4BAIXfNTvmmp9rC7JRowu+Pap2qgB8Gp/Tcc/H/ifUsxt1AQD0zDke
+GqAvrvCsVMZDVOdwnH/m4Bh2ODidIEwyy6TyBsKVBBgWCAAmAhsMFiEEsdK9E3W+
+y3hM9PjE1zz2OMU8Br4FAmD3wPEFCQXwc4cAIQkQ1zz2OMU8Br4WIQSx0r0Tdb7L
+eEz0+MTXPPY4xTwGvtQ/APwP8XxDUEogiXWIatg4A8oap242CJQjPf9IrwETGf7c
+kAEAoF/XCoJNOsSKRGPgTnVA7vD8tNQpv6BZsk4WB3ShOAfCfgQYFggAJgIbDBYh
+BLHSvRN1vst4TPT4xNc89jjFPAa+BQJf4il7BQkEjmURAAoJENc89jjFPAa+410B
+AOK0ISCDC4Rd/LLUBMSfZsLIpI1yuFnDPp+OAc0Ec9tkAQDJ1Jz/XlxCwf/gYoqT
+LXkZxGzd33QWjH2JKVZyKNF2CsJ+BBgWCAAmAhsMFiEEsdK9E3W+y3hM9PjE1zz2
+OMU8Br4FAl7ODUAFCQN6SNYACgkQ1zz2OMU8Br7vEAD/S59d+oS7f8i38eCmIwcT
+wHX3faEWHd8alhQcTWZp/gYA/12a4TWmNzBAVeMiMeLbOiU+3SX1xnGzY8StU9gZ
+V3UBwn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXbBIBgUJAjo8
+nAAKCRDXPPY4xTwGvuhAAQDvbSbrxBwmqvbEwJuprPVn9Nx/bwfWKfbS2Dqg/huO
+hAD9E1YBmxr5qiAbFJKP0O7Sj22HXX7LiALyf8eORehJPAjCfgQYFggAJhYhBLHS
+vRN1vst4TPT4xNc89jjFPAa+BQJcks9qAhsMBQkBHMQAAAoJENc89jjFPAa+4kMB
+AILG9rwrhcPSAvay4XDgJ8SqDHm8AsKA3E7zirxVq5X9AQCNlXB+RWh7yX61utkJ
+rziBqQti/j3vHKX6c3yU4FLNCw==
+=vZ4G
 -----END PGP PUBLIC KEY BLOCK-----
diff --git a/mingw-libidn2.spec b/mingw-libidn2.spec
index 6e98cf7..01a9bb1 100644
--- a/mingw-libidn2.spec
+++ b/mingw-libidn2.spec
@@ -3,12 +3,16 @@
 Summary:        MinGW Windows Internationalized Domain Name 2008 support library
 Name:           mingw-libidn2
 Version:        2.3.8
-Release:        4%{?dist}
+Release:        5%{?dist}
 License:        (GPL-2.0-or-later OR LGPL-3.0-or-later) AND GPL-3.0-or-later
 URL:            https://www.gnu.org/software/libidn/#libidn2
 
 Source0:        https://ftp.gnu.org/gnu/libidn/libidn2-%{version}.tar.gz
 Source1:        https://ftp.gnu.org/gnu/libidn/libidn2-%{version}.tar.gz.sig
+# Refreshed from keyservers/WKD (same keys, newer signatures):
+#   sq --home=none network search --output - B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE | sq keyring filter --experimental --cert B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE --output update.pgp
+#   sq keyring merge --output B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE.new B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE update.pgp
+#   mv B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE.new B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE
 Source2:        https://keys.openpgp.org/vks/v1/by-fingerprint/B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE
 
 BuildArch:      noarch
@@ -21,7 +25,7 @@ BuildRequires:  mingw64-filesystem >= 95
 BuildRequires:  mingw64-gcc
 BuildRequires:  mingw64-binutils
 
-BuildRequires:  gnupg2
+BuildRequires:  openpgpverify
 BuildRequires:  gcc
 BuildRequires:  make
 BuildRequires:  pkgconfig
@@ -73,7 +77,7 @@ Static version of the MinGW Windows IDN 2008 library.
 %?mingw_debug_package
 
 %prep
-%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %setup -q -n libidn2-%{version}
 
 %build
@@ -128,6 +132,9 @@ find $RPM_BUILD_ROOT -name "*.la" -delete
 %{mingw64_libdir}/libidn2.a
 
 %changelog
+* Fri Oct 23 2026 Jakub Jelen <jjelen@redhat.com> - 2.3.8-5
+- Verify upstream source signatures with openpgpverify
+
 * Thu Jul 16 2026 Fedora Release Engineering <releng@fedoraproject.org> - 2.3.8-4
 - Rebuilt for https://fedoraproject.org/wiki/Fedora_45_Mass_Rebuild
 
-- 
2.56.0

